src/platform/site-wide/va-footer/components/LanguageSupport.jsx

```diff
@@ -11,24 +12,19 @@ function LanguagesListTemplate({ langSelected }) {
     <ul>
       {[
         {
-          onThisPage: 'En esta página',
           label: 'Español',
-          suffix: '-esp/',
           lang: 'es',
-          href: '',
+          href: '/asistencia-y-recursos-en-espanol',
         },
         {
-          suffix: '-tag/',
           label: 'Tagalog',
-          onThisPage: 'Sa pahinang ito',
           lang: 'tl',
-          href: '/coronavirus-veteran-frequently-asked-questions-tag',
+          href: '/tagalog-wika-mapagkukunan-at-tulong',
```

Comment:
i already added these hrefs in another PR... something weird is going on in git.....
https://github.com/department-of-veterans-affairs/vets-website/pull/17383/files
cc: @kelsonic for a sanity check